website/src/components/Tasks/Task/Task.tsx

```diff
@@ -186,7 +186,7 @@ export const Task = () => {
         show={taskStatus.mode === "DEFAULT_WARN"}
         title={t(getTypeSafei18nKey(`${taskInfo.id}.unchanged_title`)) || t("default.unchanged_title")}
         message={t(getTypeSafei18nKey(`${taskInfo.id}.unchanged_message`)) || t("default.unchanged_message")}
-        continueButtonText={"Continue anyway"}
+        continueButtonText={t(getTypeSafei18nKey(`${taskInfo.id}.continue_anyway`)) || t("default.continue_anyway")}
```

Comment:
Why do you need to fallback to "default.continue_anyway" here?

Reply:
I copied the schema of the two previous strings, `title` and `message`. If I understand the code correctly, the idea is that there might be new `taksInfo.id` that doesn't have a translation and, then, the default one is used.

Reply:
ah make sense

Reply:
Instead of the `||` syntax we can use the default option instead https://www.i18next.com/translation-function/essentials#passing-a-default-value @GuilleHoardings @notmd

Reply:
Good idea. I didn't know about that. I'll try it and make a pull request.